### DISCUSSION TITLE
Directory naming

### DISCUSSION THREAD

**Discussion by adamkorynta** [opendcs/rest_api#56] [General]
Currently the gradle subproject directories are as follows:

./rest_api - for the REST API war (odcsapi.war)
./web_client - for the web client war (odcs-web-client.war)
./odcsapi-jetty-standalone - for the embedded jetty tarball installer REST API (odcsapi-jetty-standalone.tar.gz)
./odcs-web-clientjetty-standalone - for the embedded jetty tarball installer web client (odcs-web-client-jetty-standalone.tar.gz)

Gradle doesn't particularly like directories being named differently than the artifacts that it produces so I'd like to propose renaming these directories:

./opendcs-rest-api - produces opendcs-rest-api.war
./opendcs-web-client - produces opendcs-web-client.war
./opendcs-rest-api-jetty - produces opendcs-rest-api-jetty.tar.gz
./opendcs-web-client-jetty - produces opendcs-web-client-jetty.tar.gz

Please offer any suggestions for repository and artifact clarity.

**Comment by MikeNeilson** [comment]
i think that looks reasonable to me. though one thing i'd point out is that perhaps eventual we won't have that "client-jetty" as we can just package the client and api jars into single instance.

(though maybe we need it for now until that change.)

**Comment by adamkorynta** [comment]
I'm going to turn that into an issue

**Comment by adamkorynta** [comment]
Any input on `odcs-` prefix vs `opendcs-` prefix?

**Reply by rma-psmorris** [reply]
I would stick with opendcs as that is what the software is known as and odcs can lead to confusion

**Reply by MikeNeilson** [reply]
Yes, our branding is pretty much "OpenDCS-".
